tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesTokenInterceptor.java

```diff
@@ -55,7 +60,11 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
     }
 
     private boolean checkClientHostAddress(HttpServletRequest request) {
-        //stub
-        return false;
+        if (StringUtils.isNotEmpty(ipRange)) {
+            IpAddressMatcher ipAddressMatcher = new IpAddressMatcher(ipRange);
```

Comment:
move IpAddressMatcher ipAddressMatcher as a field in order to don't create it here every time.
In order to do that you need to move `@Value("${security.allowed.client.ip.range}") private String ipRange;` as constructor parameter, it also requires changes in `RestConfiguration` class, you will need to make `TesTokenInterceptor` as @Component and inject it to `RestConfiguration` and in the `RestConfiguration` pass it to `addInterceptors` methods

Reply:
after that you can return to short version of this method
```
return ipAddressMatcher != null && ipAddressMatcher.matches(request);
```